src/wallet/bdb.h

```diff
@@ -205,6 +206,7 @@ class BerkeleyBatch : public DatabaseBatch
     bool WriteKey(DataStream&& key, DataStream&& value, bool overwrite = true) override;
     bool EraseKey(DataStream&& key) override;
     bool HasKey(DataStream&& key) override;
+    bool ErasePrefix(Span<std::byte> prefix) override;
```

Comment:
Perhaps there's some nuance that I'm missing, but it seems strange to have this be `Span<std::byte>` when the rest of the methods take `DataStream&&`. I changed the `ErasePrefix` method to also take `DataStream&&` as an argument and was able to compile and pass the tests. Seems like it would be preferable to match the other methods unless you have a reason to prefer `Span<std::byte>`?

Reply:
gotcha, thanks for the explanation!